### PR TITLE
Update to March 2018 VM

### PR DESCRIPTION
**Pull request by mike-yesware**
Update references to the latest VM image.

### CODE REVIEW

**Comment by mike-yesware** [comment]
Please review but don't merge until VM is uploaded.

File: sites/en/workshop/command_line_2016.deck.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Get thee to a terminal. *(Hint: don't type the "$")*
 1. `pwd` *print working directory == where are we?*
   * `/vagrant`
 1. `ls` *list contents == see what's here*
-  * `railsbridgevm-2017-01.box Vagrantfile`
+  * `railsbridgevm-2018-03.box Vagrantfile`
```
Comment (mike-yesware) [inline sites/en/workshop/command_line_2016.deck.md:83]: Er, I just did a find/replace. Is this doc intended to be immortalized as it was in 2016?

Comment (mike-yesware) [reply sites/en/workshop/command_line_2016.deck.md:83]: Nope, the last PR that updated these also touched this. Carry on!

**Comment by mike-yesware** [comment]
Spec failure seems unrelated and something we should fix in another PR? See https://github.com/railsbridge/docs/issues/479

File: sites/en/workshop/command_line_2016.deck.md (hunk 1 above)
Comment (decklin) [reply sites/en/workshop/command_line_2016.deck.md:83]: This will be the final filename and is OK to merge now imho.

**Comment by decklin** [comment]
I'll let someone who is more familiar on the spec thing give the OK on that but content-wise 👍

**Review by edwardloveall — APPROVED** [review]
The test thing is an annoying dependency issue. I know what the problem is (the developer of the "wrong" gem hasn't pushed a new version to rubygems) but it's unrelated to this.

**Pull request merged**